config/initializers/algoliasearch.rb

```diff
@@ -1,3 +1,19 @@
+ALGOLIA_PRODUCTION_APP_ID = 'QM2YE0HRBW'
```

Comment:
Shouldn't this be an env variable?

Reply:
Given that the intent is to prevent us from accidentally reindexing the production app index from another environment, that that key is safe to publicly expose, and that we're unlikely to change the production app without advanced planning (would have to change the billing details) I felt like hardcoding it was the safer, more intention-revealing way to go with this.  Plus, if we put it in an ENV variable, then we've got yet another to manage across multiple environments :(